CatProducer/python/patTools/egmVersionedID_cff.py

```diff
@@ -8,7 +8,7 @@ def enableElectronVID(process):
     electron_ids = [
         'cutBasedElectronHLTPreselecition_Summer16_V1_cff',
         'cutBasedElectronID_Summer16_80X_V1_cff',
-        'heepElectronID_HEEPV70_cff',
+        'heepElectronID_HEEPV60_cff',
```

Comment:
is V60 correct?

Reply:
who is using HEEP and provide proper recipe?
Still the default CMSSW is pointing to V60, additional recipe is needed to enable V70.

Reply:
i think @jalmond

Reply:
I'll roll back this change to use HEEP70